### PR TITLE
Fixes: #1016. Use Origin Server and User Agent

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

As `priority` header is end-to-end, where applicable
I used "origin server" and "user agent".

Fixes: #1016 

## Notes

There might be other occurrencies to replace. Let me know if the current ones are correct.

### CODE REVIEW

File: draft-ietf-httpbis-priority.md [hunk 1]
```diff
@@ -52,12 +52,12 @@ by the retrieval of a CSS file that the document refers to.  In contrast, inline
 images do not block rendering and get drawn incrementally as the chunks of the
 images arrive.
 
-To provide meaningful representation of a document at the earliest moment, it is
+To provide meaningful presentation of a document at the earliest moment, it is
```
Comment (ioggstream) [inline draft-ietf-httpbis-priority.md]: `representation` is an http keyword. I'd use "presentation" or something similar.

File: draft-ietf-httpbis-priority.md [hunk 2]
```diff
@@ -471,9 +473,9 @@ PRIORITY_UPDATE on wrong stream, a PRIORITY_UPDATE with an invalid ID, etc.
 It is not always the case that the client has the best understanding of how the
 HTTP responses deserve to be prioritized. For example, use of an HTML document
 might depend heavily on one of the inline images. Existence of such
-dependencies is typically best known to the server.
+dependencies is typically best known to the origin server.
 
-By using the "Priority" response header, a server can override the
+By using the "Priority" response header, an origin server can override the
```
Comment (kazuho) [inline draft-ietf-httpbis-priority.md]: I am not sure if we should use "origin" here. Assuming that we would not forbid an intermediary from setting the priority header field, I think just saying "server" (or maybe "upstream") might be sufficient.

The same goes to the proposed change from "client" to "user agent."

Comment (ioggstream) [reply draft-ietf-httpbis-priority.md]: > we would not forbid an intermediary from setting the priority header field
I agree.

I interpreted the end-to-end section as "UA -> Origin Server" https://github.com/kazuho/http-extensions/blob/master/draft-ietf-httpbis-priority.md#why-use-an-end-to-end-header-field

:)

File: draft-ietf-httpbis-priority.md [hunk 3]
```diff
@@ -487,7 +489,7 @@ For example, when the client sends an HTTP request with
 priority = u=4, i=?1
 ~~~
 
-and the origin responds with
+and the origin server responds with
```
Comment (ioggstream) [inline draft-ietf-httpbis-priority.md]: @kazuho here and elsewhere "origin server" disambiguates from HTTP "origin" RFC6454 https://www.rfc-editor.org/rfc/rfc6454.html

**Comment by ioggstream** [comment]
We can probably close :)

**Pull request closed without merging**